File: app/src/main/java/com/cookplan/shopping_list/total_list/TotalShoppingListPresenter.java
package com.cookplan.shopping_list.total_list;

import com.cookplan.models.Ingredient;
import com.cookplan.models.ShopListStatus;
import com.cookplan.shopping_list.ShoppingListBasePresenter;

import java.util.List;

/**
 * Created by DariaEfimova on 24.03.17.
 */

public interface TotalShoppingListPresenter extends ShoppingListBasePresenter {

    public void changeShopListStatus(Ingredient ingredient, ShopListStatus newStatus);

    public void deleteIngredients(List<Ingredient> ingredients);
}

File: app/src/main/java/com/cookplan/shopping_list/total_list/TotalShoppingListPresenter.kt
package com.cookplan.shopping_list.total_list

import com.cookplan.models.Ingredient
import com.cookplan.models.ShopListStatus
import com.cookplan.shopping_list.ShoppingListBasePresenter

/**
 * Created by DariaEfimova on 24.03.17.
 */

interface TotalShoppingListPresenter : ShoppingListBasePresenter {

    fun changeShopListStatus(ingredient: Ingredient, newStatus: ShopListStatus)

    fun deleteIngredients(ingredients: List<Ingredient>)
}
